server/plugin.go

```diff
@@ -206,3 +210,117 @@ func (p *Plugin) sendDirectMessage(userID string, message string) error {
 	_, err = p.API.CreatePost(post)
 	return err
 }
+
+func (p *Plugin) setUserStatus(userID string, meetingID int, meetingEnd bool) error {
+	storedStatusPref, appErr := p.API.KVGet(fmt.Sprintf("%v_%v", changeStatusKey, userID))
+	if appErr != nil {
+		p.API.LogDebug("Could not get stored status preference from KV ", appErr)
+	}
+
+	if storedStatusPref == nil {
+		err := p.sendStatusChangeAttachment(userID, p.botUserID, meetingID)
+		if err != nil {
+			p.API.LogDebug("could not send status change attachment ", "error", err)
+		}
+	}
```

Comment:
Are we supposed to return in this if block?

Reply:
We can return here but the function would anyway return in the next statement. I will add return here to be clear.